usecases/traverser/explorer.go

```diff
@@ -481,36 +483,38 @@ func (e *Explorer) crossClassVectorFromModules(ctx context.Context,
 	return nil, errors.New("no modules defined")
 }
 
-func ExtractDistanceFromParams(params GetParams) float64 {
+func ExtractDistanceFromParams(params GetParams) (distance float64) {
 	if params.NearVector != nil {
-		return params.NearVector.Distance * 2
+		distance = params.NearVector.Distance
+		return
 	}
 
 	if params.NearObject != nil {
-		return params.NearObject.Distance * 2
+		distance = params.NearObject.Distance
+		return
 	}
 
 	if len(params.ModuleParams) == 1 {
-		return extractDistanceFromModuleParams(params.ModuleParams)
+		distance = extractDistanceFromModuleParams(params.ModuleParams)
 	}
 
-	panic("extractDistance was called without any known params present")
+	return
 }
 
-func ExtractCertaintyFromParams(params GetParams) float64 {
+func ExtractCertaintyFromParams(params GetParams) (certainty float64) {
 	if params.NearVector != nil {
-		return params.NearVector.Certainty
+		certainty = params.NearVector.Certainty
 	}
 
 	if params.NearObject != nil {
-		return params.NearObject.Certainty
+		certainty = params.NearObject.Certainty
 	}
```

Comment:
no `return`s here? distance have ones.

Reply:
From a business logic perspective, I think it should be impossible that multiple `near<Media>` parameters are set, so we should only ever enter a single `if` block. But agreed, it might be cleaner to have explicit returns after each statement?

Reply:
If added the return statements.